src/app/routes.tsx

```diff
@@ -156,6 +161,12 @@ const deviceManagementRoutes = (experimentalFeatures?: boolean): ExtendedRouteOb
       },
     ],
   },
+  {
+    path: '/devicemanagement/manage/:deviceId',
```

Comment:
I think this should be just 
```suggestion
    path: '/manage/:deviceId',
```

Reply:
My bad, I'll fix it